app/registries/foreman/plugin.rb

```diff
@@ -608,6 +609,12 @@ def extend_preload_scopes(model, scopes)
       (@preload_scopes[model.to_s] << scopes).flatten!.uniq!
     end
 
+    def register_template_dirs(dirs = [])
```

Comment:
```suggestion
    def register_template_dirs(dirs)
```

Since we expect **some** dirs anyway.

Not a request, but just an observation. It seems we have a bunch of plugin API method, but we don't much tell how to use them and what is actually expected as the value. I mean, here it would be nice to say what is actually expected, since I'm not sure how'd we verify/validate path without templates due to misspell or wrongly supplied string, e.g. `"app/views/temdir"` vs `"#{Plugin::Engine.root}/app/views/temdir"`  :/